packages/app/src/app/components/CodeEditor/Monaco/index.tsx

```diff
@@ -239,18 +310,17 @@ class MonacoEditor extends React.Component<Props, State> implements Editor {
     this.addKeyCommands();
     import(/* webpackChunkName: 'monaco-emmet' */ './enable-emmet').then(
       enableEmmet => {
-        enableEmmet.default(editor, monaco, {});
+        enableEmmet.default(editor, monaco);
```

Comment:
I think this one should have the `, {}`

Reply:
There is no third argument looking at `enableEmmet` function :)

Reply:
Ah lol, sorry! My bad 😄